### ISSUE TITLE
Feature request: List available licenses & ignore license case

### ISSUE THREAD

**Issue by eosrei** [tylucaskelley/licenser#4]
I incorrectly guessed the GPL would be called `gpl` so the command failed:
```
$ licenser -l gpl -e user@example -n "My Name" -p "My Project"
fatal: license gpl does not exist
$ licenser -l GPL -e user@example -n "My Name" -p "My Project"
warning: filetypes object missing from ~/.licenser.json
```

Maybe just list the available licenses when running the command?
```
$ licenser
usage: licenser [-h] -l LICENSE -e EMAIL -n NAME -p PROJECT [--no]
licenser: error: argument -l is required
Available licenses:
* MIT (mit)
* New BSD (newbsd)
* GNU GPL v3.0 (gplv3)
* Apache 2.0 (apache2)
* Mozilla 2.0 (mozilla2)
```

**Comment by tylucaskelley** [comment]
good idea, will implement when i get a chance. feel free to submit a pull request as well!

**Comment by tylucaskelley** [comment]
Fixed w/ version 2.0.2
